tests/dummy/app/templates/application.hbs

```diff
@@ -1,3 +1,5 @@
-<h2 id="title">Welcome to Ember</h2>
+<h2 id="title">Ember Plaid Link Demo</h2>
 
-{{outlet}}
\ No newline at end of file
+{{#plaid-link id="link-plaid-button" action=(action "action") onSuccess=(action "nextStep") onOpen=(action "openedLink") onExit=(action "exited")}}
```

Comment:
this will blow up immediately because these actions aren't defined on a controller, right?

Reply:
It shouldn't blow up immediately because I have a check in my actions in my addon component to make sure these are passed in. This was necessary because I wanted to write an action for onLoad (for completeness), but I didn't want to implement it in my consuming app (we have no use for that). See https://github.com/ajcolter/ember-plaid-v2/pull/1/files#diff-afc8c80e4434aa14c5b17feaa87e15d6R58 as an example.